eng/ILLink.Substitutions.Resources.template

```diff
@@ -0,0 +1,9 @@
+<linker>
+  <assembly fullname="{AssemblyName}">
+    <!-- System.Resources.UsingResourceKeys removes resource strings and instead uses the resource key as the exception message -->
+    <resource name="{StringResourcesName}.resources" action="remove" feature="System.Resources.UsingResourceKeys" featurevalue="true" />
```

Comment:
Nit: Do we use "...ing" in the names of any other features? Should this rather be `UseResourceKeys` ?